src/components/pages/Home/Home.tsx

```diff
@@ -20,6 +21,7 @@ const Home = () => {
     <OutputWindow>
       <Header />
       {(isFetching || chatbotResponse) && <SubmittedQuestion message={input} />}
+      {isFetching && <LoadingSpinner />}
       <div className="h-full">
```

Comment:
Could move the spinner inside this h-full, then position it in the centre of it -> the example questions and chatbot response won't be rendered, so there shouldn't be any clash